features/support/simplecov_setup.rb

```diff
@@ -0,0 +1,12 @@
+# @note this file is loaded in env.rb to setup simplecov using RUBYOPTs for child processes
+
+require 'simplecov'
+
+require 'pathname'
+
+root = Pathname(__FILE__).expand_path.parent.parent.parent
```

Comment:
I would prefer this here:

`File.expand_path('../../../', __FILE__)`

So you don't need to require 'pathname'.